### PR TITLE
refactor, chore(egen): Replaces K80 GPU with T4, package version updates, pre-built Docker container image for prediction update, other corrections from template

### PR DESCRIPTION
**Pull request by SumanthKasula99**
This PR consists of the following updates:

1. Replaces K80 GPU with Tesla T4 GPU in markdown section,.
2. Updates package versions.
3. Updates pre-built Docker container image for prediction to 2.13.
4. Refactors the notebook according to the notebook template (removes boilerplate elements, renames REGION with LOCATION and re-structures the sections).
5. Adds link for Colab enterprise.
6. Removes the usage of future tense(will) according to the guidelines.
7. Updates gcr to artifact registry.

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/pipelines/get_started_with_hpt_pipeline_components.ipynb [hunk 1]
```diff
@@ -631,11 +598,11 @@
         "\n",
         "The files `setup.cfg` and `setup.py` are the instructions for installing the package into the operating environment of the Docker image.\n",
         "\n",
-        "The file `trainer/task.py` is the Python script for executing the custom tuning job. *Note*, when we referred to it in the worker pool specification, we replace the directory slash with a dot (`trainer.task`) and dropped the file suffix (`.py`).\n",
+        "The file `trainer/task.py` is the Python script for executing the custom tuning job. *Note*, when referred to it in the worker pool specification, you replace the directory slash with a dot (`trainer.task`) and dropped the file suffix (`.py`).\n",
```
Comment (kittyabs) [inline notebooks/official/pipelines/get_started_with_hpt_pipeline_components.ipynb:601]: - Shouldn't this: "*Note*, when referred to it in the worker pool specification, you replace the directory slash with a dot (`trainer.task`) and dropped the file suffix (`.py`),"
- be: "Note*: When `trainer/task.py` is referred to in the worker pool specification, the directory slash is replaced with a dot and the file suffix (`.py`) is dropped (`trainer.task`)."

File: notebooks/official/pipelines/get_started_with_hpt_pipeline_components.ipynb [hunk 2]
```diff
@@ -919,49 +961,62 @@
...
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "build_container:training"
+        "id": "a6ad28220418"
       },
       "source": [
-        "Next, build the container."
+        "Then, build the container and push it to Artifact Registry by running the following:"
```
Comment (kittyabs) [inline notebooks/official/pipelines/get_started_with_hpt_pipeline_components.ipynb:975]: Should: "build the container and push it to Artifact Registry by running the following"
Be: "build the container and push it to an Artifact Registry repository by running the following".  ??

**Review by kittyabs — COMMENTED** [review]
some suggested edits

**Comment by katiemn** [comment]
Please address @kittyabs suggested edits

**Review by katiemn — APPROVED** [review]

**Pull request merged**